src/components/MakeQuizButton.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <RouterLink to="/quiz/create">
+    <div id="make-quiz-card">
+      <h1>Create Quiz</h1>
+    </div>
+  </RouterLink>
+</template>
+
+<script setup>
+import { useTheme } from "../stores/theme";
+
+const theme = useTheme();
+</script>
+
+<style scoped>
```

Comment:
Perhaps we should syle this to match the rest of the application?

Reply:
Also it is the same for dark and light mode :)

Reply:
I tried different colors from the theme store, but nothing really fit. I picked this because it was close enough and worked for both light and dark mode. We could make a task and restyle the button by adding more options to the theme store perhaps?

Reply:
Sounds good